packages/keyValueCache/src/ErrorsAreMissesCache.ts

```diff
@@ -0,0 +1,33 @@
+// This cache wraps a KeyValueCache and returns undefined for any errors thrown
+// by the underlying cache. You can also provide a logger to log errors to.
```

Comment:
Good idea on the logger.

Not that I'm ever consistent about this but should this be a doc comment on the class?